api/migrations/20180329105503_add-activity-cost-allocate.js

```diff
@@ -0,0 +1,14 @@
+exports.up = async knex => {
+  await knex.schema.createTable('activity_cost_allocations', table => {
+    table.increments('id');
+    table.string('name', 128);
+    table.integer('share');
```

Comment:
So, my sense was that the "off-the-top" flag was for "other funding sources" table (which we don't have yet) which is separate from the "cost allocation" pie chart.

I chose integer because i was thinking that there wouldn't be fractional percentages, but maybe that's a bad assumption